decoder_test.go

```
@@ -2668,6 +2669,131 @@
+	}
+
+	// Create slice for file headers
+	fileHeaders := map[string][]*multipart.FileHeader{
+		"d.f":  {dummyFile, dummyFile2},
+		"d.f2": {dummyFile2, dummyFile3},
+		"d.f3": {dummyFile, dummyFile2, dummyFile3},
+		"g":    {dummyFile, dummyFile2},
+	}
+
+	decoder := NewDecoder()
+	decoder.Decode(&s, data, fileHeaders)
```

golangci-lint (1.62.2)

Error return value of `decoder.Decode` is not checked

(errcheck)